app/templates/play/ladder/ladder.jade

```diff
@@ -162,6 +162,18 @@ block content
           else if !me.isTeacher() && !me.isAnonymous() && !me.isRegisteredForAILeague() && view.isAILeagueArena()
             a.btn.btn-illustrated.btn-block.btn-lg.btn-primary.register-for-ai-league-button(href="/league?registering=true")
               span(data-i18n="league.register_for_ai_league")
+    else if view.tournamentEnd
+      div#ladder-action-columns.row
+        if !features.china
```

Comment:
Do we not want China players to see either play option or spectate? If not, then maybe we get rid of the whole block by moving this up a line, rather than having an empty `ladder-action-columns.row`?

Reply:
oh i mean to let china players to see spectate. maybe i missed some tab.